online_classroom/src/main/java/com/sps/team2/classes/Datastore.java

```diff
@@ -7,7 +7,7 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.FilterOperator;
 
-import java.util.*;
+import java.util.ArrayList;
 
 /**
  * Provides access to the data stored in Datastore.
```

Comment:
Seems like there are several servlets want to use this object to save file. But looks like we only need on object of this class.
A way to provide only one instance for everyone who need this instance is to make this class as singleton.
First make its constructor as private.
private Datastore() {...}
Second make an public API to let everyone access the object.
private static Datastore sDatastore;
public static synchronized Datastore getDatastore() {
if (sDatastore == null) {
    sDatastore = new Datastore();
}
return sDatastore;
}
The other place can just call this api to get this instance, just like how you use DatastoreServiceFactory.getDatastoreService();